cmd/cli/uninstall_mesh_test.go

```diff
@@ -196,11 +201,9 @@ func TestUninstallCmd(t *testing.T) {
 				mem.SetNamespace(test.meshNamespace)
 			}
 
-			if test.meshExists {
```

Comment:
I'd lean towards keeping this the way it was since with this change, there's nothing stopping you from defining a test case with a mesh name that isn't `testMeshName` and `meshExists: true` (or vice versa) which won't actually be the case if whether or not the mesh actually exists is based solely on the mesh name.